robotology/speech
# Svox speech

Thanks to @apaikan we have now a very light package in Linux for **speech synthesis** that can be run as a **yarp plugin** and exposes some nice features, among which I like to point out the **multi languages support** and the possibility to tune the **voice pitch and speed**.

##### Build
- Go in `svox-speech` directory and build according to the instructions you'll find therein.

##### Test
- See whether everything is correctly in place:
    - `yarp plugin --list` and `yarpdev --list` should list down `speech` as a plugin and/or device.
- Run: `yarpdev --device speech`
- Run: `iSpeak --package speech-dev`
- Run: `yarp connect /iSpeak/speech-dev/rpc /icub/speech:rpc`
- Test: `echo "this is a test" | yarp write ... /iSpeak`

cc @robotology/wysiwyd-developers @vtikha
